misc/net.minetest.minetest.metainfo.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>net.minetest.minetest</id>
+
+    <name>Minetest</name>
+    <summary>Multiplayer infinite-world block sandbox game</summary>
+    <summary xml:lang="de">Mehrspieler-Sandkastenspiel mit unendlichen Blockwelten</summary>
+    <summary xml:lang="es">Juego sandbox multijugador con mundos infinitos</summary>
+    <summary xml:lang="fr">Jeu multijoueur bac à sable avec des mondes infinis</summary>
+    <summary xml:lang="ja">マルチプレイに対応した、無限の世界のブロック型サンドボックスゲームです</summary>
+    <summary xml:lang="ru">Игра-песочница с безграничным миром, состоящим из блоков</summary>
+    <summary xml:lang="tr">Tek-Çok oyuncuyla küplerden sonsuz dünyalar inşa et</summary>
+
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>LGPL-2.1+ and CC-BY-SA-3.0 and MIT and Apache-2.0</project_license>
+
+    <developer id="net.minetest">
+        <name>Minetest Team</name>
+    </developer>
+
+    <supports>
+        <control>pointing</control>
+        <control>keyboard</control>
+        <control>touch</control>
+        <control>gamepad</control>
```

Comment:
Minetest doesn't have functional support for gamepads, this should be removed until #4153 is done